README.md

```diff
@@ -1,2 +1,4 @@
-# home
-Código Fonte do Site do PyLadies Ribeirão Preto
+# Pyladies RP - Website
+<img src="https://pyladies-rp.github.io/img/pyladies-rp.png" width="20%">
```

Comment:
Aqui sua ideia é adicionar a logo do PyLadies-rp? Acredito que no markdown do git isso é adicionado de uma forma diferente: https://guides.github.com/features/mastering-markdown/#examples =)

Reply:
@juhhcarmona 
Acho que o certo seria assim então:

```
# Pyladies RP - Website

![Pyladies RP Logo](https://pyladies-rp.github.io/img/pyladies-rp.png)

Código fonte do site PyLadies Ribeirão Preto : https://pyladies-rp.github.io/
```
Legal, não sabia que existia este Markdown do git.Obrigado! ;)

- O problema seria que desta forma não teria como redimensionar o logo.
Lendo achei no Stackoverflow : https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown

Eles falam que utilizando o ```<img src=" " width="20%">``` poderia estar certo.

Será que podemos confiar?

Desculpa pelo trabalho!
Obrigado!

Reply:
Imagina, trabalho nenhum! =D Bom, eu não sei, vamos descobrir? hehehe

Reply:
@jonasbp deu certinho =D Parabéns! ;)

Reply:
Oba! Adorei ajudar, hahahaah